### DISCUSSION TITLE
Backoffice: plugin based on user group permissions

### DISCUSSION THREAD

**Discussion by flower-of-the-bridges** [mia-platform/community#184] [Q&A] [answered]
Hi

Is it possible to filter backoffice plugins based on user group permissions?

For example, If I have the following file `configuration.json`:

```
"plugins": [{
    "id": "admin",
    "label": "Admin",
    "icon": "fas fa-biking",
    "integrationMode": "qiankun",
    "pluginRoute": "/admin",
    "pluginUrl": "/element-composer/",
    "props": {
      "configurationName": "admin"
    }
  }, {
    "id": "developer",
    "label": "Developer",
    "icon": "fas fa-biking",
    "integrationMode": "qiankun",
    "pluginRoute": "/developer",
    "pluginUrl": "/element-composer/",
    "props": {
      "configurationName": "developer"
    }
  }]
```

I would like a user with permissions `admin` to see only the plugin `Admin`, while a user with `developer` permission can see only the `developer` plugin. 

Thanks!

**Comment by clabene** [comment, marked as answer]
Hi @giovf96, in v1 of micro-lc you can add `aclExpression` key in your plugins config. This field lets you specify a string predicate based on the users groups, as described here: https://micro-lc.io/1.0.0/docs/core_configuration#plugin-parameters

**Reply by flower-of-the-bridges** [reply]
Hi @clabene thank you very much!
